### PR TITLE
Implement Square Root Feature

### PR DESCRIPTION
**Pull request by Sukhilnair**
Added square root feature to the Calculator class.

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -12,15 +12,12 @@ def multiply(self, a, b):
...
         return a / b
 
-# TODO: Implement the following function to calculate the square root of a number.
-
-# def square_root(self, x):
-
-# return math.sqrt(x)
-
-# You need to uncomment the above function and complete its implementation to add the square root feature.
+    def square_root(self, x):
```
Comment (sonalbatch5) [inline calculator.py:25]: Add description of this function to describe its functionality.

File: calculator.py [hunk 2]
```diff
@@ -12,15 +12,12 @@ def multiply(self, a, b):
         return a * b
 
     def divide(self, a, b):
```
Comment (sonalbatch5) [inline calculator.py]: please use better variable name e.g. a can be replaced with variable name like numerator and b can used as denominator.

**Review by sonalbatch5 — COMMENTED** [review]
Added review comments

File: calculator.py (hunk 2 above)
Comment (Sukhilnair) [reply calculator.py]: Sure will update. and push back

**Comment by Sukhilnair** [comment]
Hi @sonalbatch5 , 

I have updated the code with your suggestion. Please re-verify and let me know if it is good to merge.

**Comment by sonalbatch5** [comment]
The changes look good to me.

**Comment by sonalbatch5** [comment]
> Hi @sonalbatch5 ,
> 
> I have updated the code with your suggestion. Please re-verify and let me know if it is good to merge.

the changes look good to me.

**Pull request merged**